### PR TITLE
finished week-2 assignment of NodeJS

### PR DESCRIPTION
**Pull request by choudhary-keshav**
Pull Request Questionnaire

(1) Which Jira ticket(s) is this pull request for, and is the ticket now complete?

(2) Briefly describe your change.
I have pushed my week-2 assignment in this nodeJS project. In my week-2 assignment I have used middlewares & static, dynamic routes.

(3) What parts of the application are affected by your change?
Nothing

(4) How did you test your change? If relevant, which role(s) and/or customer configuration(s) did you use?
Tested my application on various edge cases and some common scenarios

### CODE REVIEW

**Review by aditya-s-19 — APPROVED** [review]

File: src/Week2/assignment.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//localhost:3000/?email=user2@example.com&password=password2
+//localhost:3000/
```
Comment (shreyansh-khatri) [inline src/Week2/assignment.ts]: Remove these comments

File: src/Week2/assignment.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const express = require("express");
+const fs = require("fs");
+const app = express();
+
+const authenticateUser = (req: any, res: any, next: any) => {
```
Comment (shreyansh-khatri) [inline src/Week2/assignment.ts]: Please specify return type

**Review by shreyansh-khatri — CHANGES_REQUESTED** [review]

**Review by RitikaGarg-ITT — APPROVED** [review]

**Review by shreyansh-khatri — APPROVED** [review]

**Review by shreegovinda — APPROVED** [review]

**Review by anusreesubash — APPROVED** [review]
